### DISCUSSION TITLE
An activity per customer but not project?

### DISCUSSION THREAD

**Discussion by Silver-Golden** [kimai/kimai#2762] [Questions & Answers]
Hi there,

Is it possible to assign an activity to a customer but not to a specific project?  
For example I have my College as my customer, the module code as the project and want to create an activity named Studying to all the projects (modules) under the customer (college).  
I tried to set it in the webgui but it keeps resetting.

Brendan.

**Comment by kevinpapst** [comment]
Hi Brendan, 

sorry, but that is not possible. 

The customer field in the activity-form was only a filter for the project list.
I removed the customer field from the activity edit screen with 1.15, because it caused a lot of questions like yours in the past.

Best regards
Kevin
